src/Assert.php

```diff
@@ -719,6 +721,16 @@ public static function notRegex($value, $pattern, $message = '')
         }
     }
 
+    public static function letter($value, $message = '')
```

Comment:
As discussed in the issue, letter may not be the correct name for this assertion, as it implies singular.

Reply:
I don't see that on the issue?

Perhaps either `letters()` or `unicodeLetters()`?